keel-clouddriver/src/test/resources/loadBalancers.json

```diff
@@ -0,0 +1,1357 @@
+[
+  {
+    "account": "test",
+    "name": "keeldemo-main-clb",
+    "region": "us-east-1",
+    "vpcId": "vpc-46f5a223",
```

Comment:
You might want to obfuscate real AWS IDs for potentially sensitive things like VPCs, Route53 hosted zones, as well as ELB hostnames.

Reply:
I've obscured it all now